src/ACEtk/block/LevelScatteringDistribution/src/verifySize.hpp

```diff
@@ -0,0 +1,12 @@
+static void verifySize( Iterator begin, Iterator end ) {
+
+  // there are at least 2 values in the block
```

Comment:
Is "at least" the right check?  Isn't this an "exactly" case?

Reply:
The XSS array can have "gaps" in, hence the "at least". The way the ACE file is set up also allows for gaps to appear. Technically speaking the blocks may even appear anywhere in the file since the starting index is only given as a locator (without an ending one). When generating the blocks, I probably should sort the locators first instead of assuming that they always appear in a specific order. 

Also, NJOY2016 sometimes tweaks the ACE file (I put some of these in actually) and in some cases, we would have to rearrange the XSS array and every single locator in it so I left it as a gap. MCNP doesn't even notice. NJOY2016 does signal whenever a gap is detected during locator checking.